app/src/main/java/org/gdg/frisbee/android/view/CircleTransform.java

```diff
@@ -0,0 +1,50 @@
+package org.gdg.frisbee.android.view;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapShader;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+
+import com.squareup.picasso.Transformation;
+
+/**
+ * Created by <a href="mailto:marcusandreog@gmail.com">Marcus Gabilheri</a>
+ *
+ * @author Marcus Gabilheri
+ * @version 1.0
+ * @since 7/26/15.
+ */
+public class CircleTransform implements Transformation {
```

Comment:
Hmmm. The only one that I could find was the BitmapBorderTransformation. is that in another branch?